package.json

```diff
@@ -73,7 +73,7 @@
     "graphql": "^16.8.1",
     "graphql-tag": "^2.12.6",
     "siwe": "^2.1.4",
-    "viem": "~2.21.0"
+    "viem": "~2.21.25"
```

Comment:
Updated this for any particular reason? this must be synced with the one below to be matched by any implementation and avoid wallet type incompatibility problems

Reply:
to match frontend's version

Reply:
they screwed the minors